### PR TITLE
Update bootstrap.sh

### PR DESCRIPTION
**Pull request by yanganto**
This PR is a part of https://github.com/darwinia-network/darwinia/issues/431, and also a part of preparation of workshop.

### CODE REVIEW

File: .maintain/bootstrap.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ if [[ "$1" != "--fast" ]]; then
             exit 1
         elif [ -f /etc/arch-release ]; then
             echo "Arch Linux detected."
-            sudo pacman -Syu --needed --noconfirm cmake gcc openssl-1.0 clang llvm rocksdb curl
+            sudo pacman -Syu --needed --noconfirm cmake gcc openssl-1.0 clang llvm rocksdb curl grep
```
Comment (AurevoirXavier) [inline .maintain/bootstrap.sh:21]: why not use latest openssl?

Comment (yanganto) [reply .maintain/bootstrap.sh]: Thanks, I will fix it.

File: .maintain/bootstrap.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ if [[ "$1" != "--fast" ]]; then
             exit 1
         elif [ -f /etc/arch-release ]; then
             echo "Arch Linux detected."
-            sudo pacman -Syu --needed --noconfirm cmake gcc openssl-1.0 clang llvm rocksdb curl grep
+            sudo pacman -Syu --needed --noconfirm cmake gcc openssl clang llvm rocksdb curl grep
             export OPENSSL_LIB_DIR="/usr/lib/openssl-1.0";
```
Comment (AurevoirXavier) [inline .maintain/bootstrap.sh:22]: still 1.0?

File: .maintain/bootstrap.sh [hunk 3]
```diff
@@ -18,7 +18,7 @@ if [[ "$1" != "--fast" ]]; then
...
             export OPENSSL_INCLUDE_DIR="/usr/include/openssl-1.0"
```
Comment (AurevoirXavier) [inline .maintain/bootstrap.sh:23]: still 1.0?

**Pull request merged**